src/main/java/frc/robot/subsystems/climber/ClimberSubsystem.java

```diff
@@ -15,11 +15,24 @@
     boolean activateClimber = false;
     boolean operatorHolding = false;
     boolean driverHolding = false;
+    double leftArmMoved1;
+    double rightArmMoved1;
 
     public void initialize() {
         leftArm = new WPI_TalonSRX(12);
         rightArm = new WPI_TalonSRX(8);
     }
+    public void moveArms(double leftArmMoved,double rightArmMoved)
+    {
+        leftArmMoved1 = leftArmMoved;
+        rightArmMoved1 = rightArmMoved;
```

Comment:
This is fine for the first checkin, but I don't think we need to store the last amount moved for each arm. If you have a reason for it though, by all means, go for it.